ee/dtr/index.md

```diff
@@ -6,61 +6,42 @@ keywords: registry, repository, images
-see a list of all of the software installed in your images. Additionally, you can now [review and audit jobs on the web interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
+DTR can serve as a Continuous Integration and Continuous Delivery (CI/CD) component, in the building, shipping, and running of applications.
 
 ## Availability
 
-DTR is highly available through the use of multiple replicas of all containers
-and metadata such that if a machine fails, DTR continues to operate and can be repaired.
+DTR is highly available through the use of multiple replicas of all containers and metadata. As such, DTR will continue to operate in the event of machine failure, thus allowing for repair.
 
 ## Efficiency
 
-DTR has the ability to [cache images closer to users](admin/configure/deploy-caches/index.md)
-to reduce the amount of bandwidth used when pulling Docker images.
-
-DTR has the ability to [clean up unreferenced manifests and layers](admin/configure/garbage-collection.md).
+DTR is able to reduce the bandwidth used when pulling Docker images by [caching images closer to users](admin/configure/deploy-caches/index.md). In addition, DTR can [clean up unreferenced manifests and layers](admin/configure/garbage-collection.md).
 
 ## Built-in access control
 
-DTR uses the same authentication mechanism as Docker Universal Control Plane.
-Users can be managed manually or synchronized from LDAP or Active Directory. DTR
-uses [Role Based Access Control](admin/manage-users/index.md) (RBAC) to allow you to implement fine-grained
-access control policies for your Docker images.
+As with Universal Control Plane (UCP), DTR uses [Role Based Access Control (RBAC)](admin/manage-users/index.md), which allows you to manage image access, either manually, with LDAP, or with Active Directory.
 
 ## Security scanning
 
-DTR has a built-in security scanner that can be used to discover what versions
-of software are used in your images. It scans each layer and aggregates the
-results to give you a complete picture of what you are shipping as a part of
-your stack. Most importantly, it correlates this information with a
-vulnerability database that is kept up to date through [periodic
-updates](admin/configure/set-up-vulnerability-scans.md). This
-gives you [unprecedented insight into your exposure to known security
-threats](user/manage-images/scan-images-for-vulnerabilities.md).
+A security scanner is built into DTR, which can be used to discover the versions of the software that is in use in your images. This tool scans each layer and aggregates the results, offering a complete picture of what is being shipped as a part of your stack. Most importantly, as the security scanner is kept up-to-date by tapping into a [periodically updated](admin/configure/set-up-vulnerability-scans.md) vulnerability database, it is able to provide [unprecedented insight into your exposure to known security threats](user/manage-images/scan-images-for-vulnerabilities.md).
```

Comment:
Unprecedented insight is a strong statement. I'd be cautious about using such a strong statement. 
Although I don't have any idea about the scanning market unprecedented would seem to indicate that what we are offering is the only one of it's kind when I think competitors have similar offerings particularly because we are using a vendor to provide the scanning solution. This means that competitors can use the same vendor. 

I also think that other vendors probably provide a solution of a similar quality.